landing_page/app/javascript/components/Home.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import CSS from 'csstype';
+
+type HomeProps = {
+    title:string
+    content:string
+};
+
+const h1Styles: CSS.Properties = {
+    padding: '1.0rem',
+    fontFamily: 'sans-serif'
+};
+
+const h3Styles: CSS.Properties = {
+    fontFamily: 'sans-serif',
+    color: 'lime'
+};
+
+class Home extends React.Component<HomeProps> {
+    render (){
+        return (
+            <div className="Home">
```

Comment:
no point in adding a css class if it is not actually used in your css files

Reply:
This was added in as tsx elements need to have one parent element

Reply:
But the parent element is the div right? Does it need to have a css class assigned to it?

Reply:
That's a fair point. I'll remove the class from it now